testdata/errors/types.go

```diff
@@ -7,6 +7,6 @@ func main() {
 }
 
 // ERROR: # command-line-arguments
-// ERROR: types.go:4:6: a declared and not used
+// ERROR: types.go:4:6: declared and not used: a
```

Comment:
Note this change does mean tests don't pass on Go < 1.23, though 1.19 is the minimum.

Reply:
Correct. A number of tests only work on the latest Go and LLVM versions. But I think all the tests in main_test.go for example should pass in most or all Go and LLVM versions.
And of course the smoke test pass in all supported Go and LLVM versions, because that's we're testing in CI.

You do have a point though. I've added some basic tests for the oldest supported Go/LLVM version here: https://github.com/tinygo-org/tinygo/pull/4562